exercises/react/react.py

```diff
@@ -0,0 +1,11 @@
+class Cell(object):
+    def set_value(value):
+        pass
+
+
+class Reactor(object):
+    def create_input_cell(value):
+        pass
+
+    def create_compute_cell(dependencies, updater):
```

Comment:
Presumably these three methods should have `self` as an argument?

Reply:
Yep. Thanks.